purchase_request_to_procurement/views/purchase_request.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+<data>
+
+    <!-- Purchase Request -->
+
+    <record model="ir.ui.view" id="purchase_request_form_view">
+        <field name="name">purchase.request.form (in purchase_request_to_procurement)</field>
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='picking_type_id']" position="after">
+                <field name="warehouse_id"/>
+                <field name="location_id"/>
+            </xpath>
+            <xpath expr="//field[@name='line_ids']/form//field[@name='procurement_id']" position="replace"/>
+            <xpath expr="//field[@name='line_ids']/form//notebook"
+                   position="inside">
+                <page name="Procurement Order"
+                      string="Procurement Order">
+                    <group name="procurement_infos">
+                        <field name="procurement_id" />
+                        <field name="procurement_state"/>
+                    </group>
+                </page>
+            </xpath>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="purchase_request_tree_view">
+        <field name="name">purchase.request.tree (in purchase_request_to_procurement)</field>
+        <field name="model">purchase.request</field>
+        <field name="inherit_id" ref="purchase_request.view_purchase_request_search"/>
```

Comment:
maybe this extension of the tree view can be avoided but the one of the form view with the new page in the notebook is needed at least, don't you think so?